README.md

```diff
@@ -1,64 +1,123 @@
+
+You can choose the highlighting style from:
+`none` `text` `filled` `outlined` `underlined_solid` `underlined_strippled` `underlined_squiggly`
+
+![colored text highlighting preview][image-colored-text-preview]
+
+**NOTE**: Underlined styles are available in Sublime Text 3 only.
+
+You can also turn on highlighting all colors at once. This mode has own highlighting style, so you can highlight all colors with underline and selected colors with filled rectangle.
+
+![highlight all preview][image-highlight-all]
+
+You can also enable icons, which will be shown in the gutter of a file (might not work in ST2).
 
-Just put the cursor (or multiple cursors) on the color code and select "Choose color" in context menu (or press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>C</kbd>). Select the color in a popup color picker and all color codes under your cursors will change. The change will preserve exact code format, so if you select codes "#FFF" and "rgb(255,255,255)" and "white" and choose color "#FF0000", you get codes: "#F00" and "rgb(255,0,0)" and "red". Also, works with variables.
+You can always turn off default key-bindings via main menu or settings.
 
-**Color converter usage:**
+## Contributing
+To contribute to the project, just head on over to the [Github Project][project-url].
 
-Just put the cursor (or multiple cursors) on the color code and select "Convert color" in context menu (or press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>A</kbd>). Input a color format to use and press enter. The full list of all color formats is in the plugins settings file.
-Format can be any format supported by this plugin, for example if you convert "rgb(255,255,255)" into format "hsv", you'll get "hsv(0, 0%, 100%)". Also works with named colors and variables.
+Found something that doesn't work? Let us know by opening an issue.
+Have an idea? Open up an issue and let's make it happen!
+Found a typo? Send in a pull request.
 
-Also, there is a Prev/Next color commands, triggered by <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>,</kbd> and <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>.</kbd> respectively to choose previous or next color format.
+Don't hesitate. :smile:
 
-**Less/sass/scss/stylus variables navigation:**
+ [project-url]: https://github.com/Monnoroch/ColorHighlighter
+ [package-control]: https://packagecontrol.io/installation
+ [imagemagick-url]: http://www.imagemagick.org/
 
-Just put cursor on a variable, right click on it and press "Go to variable definition" and the plugin will open it. There is also a shortcut <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>D</kbd>.
+ [image-color-picker]: images/color-picker.gif
+ [image-color-converter]: images/convert.gif
+ [image-underline-preview]: images/underline-preview.gif
+ [image-colored-text-preview]: images/colored-text-preview.gif
+ [image-gutter-dots-preview]: images/dots.gif
+ [image-go-to-definition]: images/go-to-definition.gif
```

Comment:
Did you create this GIF yourself? Is it possible to remove the bottom part with the wallpaper that is not useful? Same goes for convert.gif.

Reply:
Ping.